tests/integration_tests/test_sql_injection_protection.py

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
I would suggest that we leave only some basic integration tests here.

Most of the validator functionality is being tested in the unit tests so we don't need as many tests here.

Reply:
Should we move all the tests for sql injection be unit tests?

Reply:
No. Most should be unit tests but there should be some (2-3) integration tests testing the whole flow.

Reply:
Will this be in a new branch?
So its more clear the intents of the changes,

Reply:
We have already changed the validator and added unit tests, on this branch, so I guess it doesn't make sense to have a new branch just for that...